packages/blade/src/components/ActionList/ActionList.tsx

```diff
@@ -1,119 +1,163 @@
 import React from 'react';
+import styled from 'styled-components';
+import { getActionListContainerRole, getActionListItemWrapperRole } from './getA11yRoles';
+import { getActionListProperties } from './actionListUtils';
 import Box from '~components/Box';
 import { useDropdown } from '~components/Dropdown/useDropdown';
-import { Text } from '~components/Typography';
-import { getPlatformType, makeAccessible } from '~utils';
-
-type ActionListItemProps = {
-  title: string;
-  value: string;
-  href?: string;
-  /**
-   * Internally passed from ActionList. No need to pass it explicitly
-   *
-   * @private
-   */
-  _index?: number;
+import {
+  isReactNative,
+  makeAccessible,
+  makeSize,
+  isAndroid,
+  metaAttribute,
+  MetaConstants,
+} from '~utils';
+import { useTheme } from '~components/BladeProvider';
+
+type ActionListProps = {
+  children: React.ReactNode[];
+  surfaceLevel?: 2 | 3;
+};
+
+const getReactNativeShadow = ({
+  offsetX,
+  offsetY,
+  shadowColor,
+  blur,
+}: {
+  offsetX: number;
+  offsetY: number;
+  shadowColor: string;
+  blur: number;
+}): {
+  shadowOpacity?: '1';
+  shadowRadius?: number;
+  shadowColor?: string;
+  shadowOffset?: string;
+} => {
+  if (isReactNative()) {
+    return {
+      shadowOpacity: '1',
+      shadowRadius: isReactNative() ? undefined : blur,
+      shadowColor: isAndroid() ? undefined : shadowColor,
```

Comment:
I hope you double checked the shadow to ensure it looks as close as possible in all three platforms.

Reply:
Yup I did. Looks a bit strange on Android but don't see much options there anyway ;__;. Also this component on React Native will be replaced with BottomSheet so it's temporary so didn't spend much time to make it too accurate.

ios:
<img width="401" alt="image" src="https://user-images.githubusercontent.com/30949385/216323754-ff285c1c-e666-4c45-a0c8-9a2bfe567688.png">

android: 
<img width="554" alt="image" src="https://user-images.githubusercontent.com/30949385/216327818-385ed810-d4bc-4d21-9532-8a56a987414b.png">